system-test/bigtable.ts

```diff
@@ -1414,6 +1421,268 @@ describe('Bigtable', () => {
+            copyExpireTime.toStruct()
+          );
+        });
+        it('should copy to the same cluster with date expiry times', async () => {
+          await testWithExpiryTimes(
+            new Date(sourceExpireTimeMilliseconds),
+            new Date(copyExpireTimeMilliseconds)
+          );
+        });
+      });
+      it('should create backup of a table and copy it on another cluster of another instance', async () => {
+        const [backup, op] = await TABLE.createBackup(generateId('backup'), {
+          expireTime: sourceExpireTime,
+        });
+        {
+          await op.promise();
+          // Check the expiry time.
+          await backup.getMetadata();
+          assert.deepStrictEqual(backup.expireDate, sourceExpireTime);
+        }
+        // Create another instance
+        const instance = bigtable.instance(generateId('instance'));
+        const destinationClusterId = generateId('cluster');
+        {
+          // Create production instance with given options
+          const instanceOptions: InstanceOptions = {
+            clusters: [
+              {
+                id: destinationClusterId,
+                nodes: 3,
+                location: 'us-central1-f',
+                storage: 'ssd',
+              },
+            ],
+            labels: {'prod-label': 'prod-label'},
+            type: 'production',
+          };
+          const [, operation] = await instance.create(instanceOptions);
+          await operation.promise();
+        }
+        // Create the copy and test the copied backup
+        await testCopyBackup(
+          backup,
+          {
+            cluster: new Cluster(instance, destinationClusterId),
+            id: generateId('backup'),
+            expireTime: copyExpireTime,
+          },
+          instance
+        );
+        await instance.delete();
+      });
+      it('should create backup of a table and copy it on another cluster of the same instance', async () => {
+        const [backup, op] = await TABLE.createBackup(generateId('backup'), {
+          expireTime: sourceExpireTime,
+        });
+        {
+          await op.promise();
+          // Check the expiry time.
+          await backup.getMetadata();
+          assert.deepStrictEqual(backup.expireDate, sourceExpireTime);
+        }
+        const destinationClusterId = generateId('cluster');
+        {
+          // Create destination cluster with given options
+          const [, operation] = await INSTANCE.cluster(
+            destinationClusterId
+          ).create({
+            location: 'us-central1-b',
+            nodes: 3,
+          });
+          await operation.promise();
+        }
+        // Create the copy and test the copied backup
+        await testCopyBackup(
+          backup,
+          {
+            cluster: new Cluster(INSTANCE, destinationClusterId),
+            id: generateId('backup'),
+            expireTime: copyExpireTime,
+          },
+          INSTANCE
+        );
```

Comment:
Delete the instance here after testing?

Reply:
This instance is already deleted in https://github.com/googleapis/nodejs-bigtable/blob/d30e5133a55e1b8f269b3aa6a69b49415361ba34/system-test/bigtable.ts#L98-L115.